submit_muscat_imaging_requestgroup.py

```diff
@@ -0,0 +1,104 @@
+#!/bin/env python
+"""
+submit_muscat_imaging_requestgroup.py
+
+Submit an imaging RequestGroup for the MUSCAT instrument to be scheduled for observing.
+"""
+import requests
+
+API_TOKEN = 'PlaceApiTokenHere'  # API token obtained from https://observe.lco.global/accounts/profile/
+PROPOSAL_ID = 'PlaceProposalHere'  # Proposal IDs may be found here: https://observe.lco.global/proposals/
+
+# The target of the observation
+target = {
+    'name': 'm40',
+    'type': 'ICRS',
+    'ra': 185.55221,
+    'dec': 58.08294,
+    'epoch': 2000
+}
+
+# Constraints used for scheduling the observation
+constraints = {
+    'max_airmass': 1.6,
+    'min_lunar_distance': 30
+}
+
+# The configurations for this request. The MUSCAT instrument has extra_params within it's instrument
+# configurations for the g, r, i, z exposure times and the exposing mode. It has optical_elements groups
+# in its instrument configurations for g, r, i, z diffuser states (ON or OFF)
+# For more information on the MUSCAT instrument, look HERE
```

Comment:
the HERE should be replaced with a link to the MUSCAT getting started guide once that is live